cluster/hooks.go

```diff
@@ -106,7 +107,34 @@ func InstallMonitoring(input interface{}) error {
 	if !ok {
 		return errors.Errorf("Wrong parameter type: %T", cluster)
 	}
-	return installDeployment(cluster, helm.DefaultNamespace, pkgHelm.BanzaiRepository+"/pipeline-cluster-monitor", "pipeline-monitoring", nil, "InstallMonitoring")
+
+	const grafanaAdminUsername = "admin"
```

Comment:
Shouldn't rather this come from config?

Reply:
Now you can configure the grafana admin username.

Reply:
Should be externalized to viper/config